src/main/java/org/jacop/constraints/XneqY.java

```diff
@@ -78,6 +78,11 @@ public XneqY(IntVar x, IntVar y) {
     }
```

Comment:
Thank you for your contribution. However, I will not be able to accept any PR unless you sign the Contributors agreement. I have explained basic details in README in Contributors section.

Reply:
signed, thanks!